keysign/discover.py

```diff
@@ -42,13 +42,17 @@ def start(self):
         if self.bt_code and not self.stopped:
             # We try Bluetooth, if we have it
             log.info("Trying to connect to %s with Bluetooth", self.bt_code)
-            self.bt = BluetoothReceive(self.bt_port)
+            # We try to see if Bluetooth was imported,
+            # else we log an event of missing Pybluez.
+            try:
+                self.bt = BluetoothReceive(self.bt_port)
+            except:
```

Comment:
When Tobi suggested you to avoid a "broad catch", he meant that you should always catch the exact exception that can be raised instead of writing a generic `except`.

Reply:
I misunderstood that totally.Will Take care of it next time.